docs/TuxOwneeClass.md

```diff
@@ -0,0 +1,250 @@
+
+If you are unfamiliar with `React` components please review their [docs](http://facebook.github.io/react/). Below is the API documentation for the `createOwneeClass` method.
+
+***
+
+### <a id="React-createOwneeClass"></a>3) React.createOwneeClass [#](#React-createOwneeClass)
+Returns an `OwneeClass` based on the passed in `owneeClassProps`
+
+```javascript
+    var OwneeClass = React.createOwneeClass(owneeClassProps);
+```
+
+#### <a id="createOwneeClass-owneeClassProps"></a>Parameter - `owneeClassProps` - type: OBJECT - required [#](#createOwneeClass-owneeClassProps)
+Properties of the `OwneeClass` to be created. Expected keys:
+
+##### <a id="owneeClassProps-render"></a>Property - `owneeClassProps.render` - type: FUNCTION - required [#](#owneeClassProps-render)
+As with any other `React` class, a [render](#http://facebook.github.io/react/docs/component-specs.html#render) method is required. This method is responsible for returning a `React Element` which can be rendered onto the DOM.
+
+```javascript
+    owneeClassProps.render = function () {
+      return (
+        <div>
+          <h1>hello world</h1>
+        </div>
+      );
+    };
+```
+
+##### <a id="owneeClassProps-LIFECYCLE_AND_SPECS"></a>Methods/Properties - `owneeClassProps.LIFECYCLE_AND_SPECS` - type: FUNCTIONs - optional [#](#owneeClassProps-LIFECYCLE_AND_SPECS)
+As with any other `React` class, the `owneeClassProps` can optionally include `React` [lifecycle methods and specs](#http://facebook.github.io/react/docs/component-specs.html).
+
+```javascript
+    owneeClassProps.componentDidMount = function () {
+      roomActions.get();
+    };
+```
+
+##### <a id="owneeClassProps-nearestOwnerProps"></a>Property - `owneeClassProps.nearestOwnerProps` [#](#owneeClassProps-nearestOwnerProps)
+Allows the `Ownee` to access the `nearestOwnerProps` on the closest `OwnerClass`.
+nearestOwnerProps - should be static, if you need to pass in dynamic props use standard method (works well for an example)
+
+**NOTE** `nearestOwnerProps` should be treated as immutable just like standard component props. Thus, you should never modify the `nearestOwnerProps` object. It is designed to store static methods and data.
+
+##### <a id="owneeClassProps-nearestOwnerPropTypes"></a>Property - `owneeClassProps.nearestOwnerPropTypes` - type: OBJECT - optional [#](#owneeClassProps-nearestOwnerPropTypes)
+Allows you to perform prop validation on the `nearestOwnerProps` to this `OwneeClass` component. For the full list of available properties that can be checked, please look at the [Reusable Components](http://facebook.github.io/react/docs/reusable-components.html) documenation.
```

Comment:
Change end of first sentence to: ...on the `nearestOwnerProps` object on this component.

Corrects spelling of `documentation`.